lib/uploadcare/concerns/unthrottleable.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Uploadcare
+  # This module lets clients send request multiple times if request is throttled
+  module Unthrottleable
```

Comment:
the name is misleading :)
this doesn't make client unthrottable, on the contrary, it makes it handle throttling in responsible manner.

```suggestion
  module ThrottleHandler
```

Reply:
Renamed